ui/apps/dashboard/src/components/Forms/Input.tsx

```diff
@@ -52,7 +52,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>((props, ref) => {
         placeholder={placeholder}
         value={props.value}
         className={cn(
-          'border border-slate-300 text-sm leading-none placeholder-slate-500 shadow outline-2 outline-offset-2 outline-indigo-500 transition-all focus:outline',
+          'border border-slate-400 text-sm leading-none placeholder-slate-400 shadow outline-2 outline-offset-2 outline-indigo-500 transition-all focus:outline',
```

Comment:
I'd not change these colors now, as this component is used next to other elements with similar border colors - and the DateInputButton is using similar border colors so they would better match. 
<img width="535" alt="Screenshot 2024-05-31 at 16 15 36" src="https://github.com/inngest/inngest/assets/16758464/1533c2f2-2a50-402e-99ec-88598d4c6daf">

Reply:
👍